ntbs-service/Services/NotificationService.cs

```diff
@@ -349,5 +350,14 @@ public async Task DenotifyNotification(int notificationId, DenotificationDetails
             notification.NotificationStatus = NotificationStatus.Denotified;
             await UpdateDatabase(AuditType.Denotified);
         }
+
+        public async Task DeleteNotification(int notificationId, string deletionReason)
+        {
+            var notification = await repository.GetNotificationAsync(notificationId);
+            notification.DeletionReason = deletionReason;
+
+            notification.NotificationStatus = NotificationStatus.Deleted;
+            await UpdateDatabase(AuditType.Deleted);
```

Comment:
Due to DeletionReason and NotificationStatus being columns on the notification entity the audit database should pick up these changes as updates however I need to confirm this

Reply:
Can test it locally, poke me when you're back and I'll show ya how :)